### DISCUSSION TITLE
sidebar/sanfene/shejimoshi

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#272] [Announcements]
# sidebar/sanfene/shejimoshi

下载次数超 1 万次，1 万字 1 张手绘图，详解 1 道 设计模式 面试高频题（让天下没有难背的八股），面渣背会这些 设计模式 八股文，这次吊打面试官，我觉得稳了（手动 dog）。

https://www.javabetter.cn/sidebar/sanfene/shejimoshi.html

**Comment by LVXUEYANGTiger** [comment]
学习了

**Comment by hachinekooo** [comment]
mark
